pkg/crypto/apple.go

```diff
@@ -91,19 +93,23 @@ pyQPbp8orlXe+tA8JA==
 -----END CERTIFICATE-----
 `
 
+// extracted public key from appleiPhoneDeviceCAPEM
+var appleiPhoneDeviceCAPublicKey = &rsa.PublicKey{
```

Comment:
I'd like to keep this within the single X.509 PEM that's already in the source. If optimization was the concern we could add an `init()` function that does the parsing just once at startup time (though I generally dislike them). I know you test for equality in the test but I like the single source of truth better. Thoughts? Could also wrap this into an interface with its own initializer as well.

Reply:
I thought the test was sufficient, but I can see how it seems a little "dirty".

I pushed a commit that replaces the handcrafted public key with a "mustParsePublicKey" function. This way, we're still only parsing it once, but we have the single source of truth. Does that work for you?

Reply:
Yeah this looks good!

I don't think its "dirty" per se — I just dislike multiple sources of truth. Like if we wanted to update that cert or whatever we only have one place we need to do it. (Though, honestly, if Apple did update that cert we'd probably have to go do something else yet again heh).